### DISCUSSION TITLE
Use local venv with .python-version

### DISCUSSION THREAD

**Discussion by jborbik** [pyenv/pyenv#2663] [Q&A] [answered]
Is it possible to point to the locally installed venv with .python-version file so that it is automatically activated over pyenv?

**Comment by native-api** [comment, marked as answer]
Pyenv needs an inventory of environments to select from in a known place -- so that it knows what to pick when you say "use installation/environment 'foo'".

So if you symlink your environment into `$PYENV_ROOT/versions`, you should be able to select it with Pyenv like any other version.

**Reply by jborbik** [reply]
Ah, makes sense. Thank you!
